From dfee9d396430d57efcad9c8d6cccb0d40bf87dae Mon Sep 17 00:00:00 2001
From: Yang Shi <yang.shi@windriver.com>
Date: Mon, 9 Aug 2010 14:07:13 +0800
Subject: [PATCH 05/14] powerpc: mpc85xx: Add DS3232 RTC device node for P1022DS board

DS3232 RTC resides on I2C bus #2 0x68, add the device node in P1022DS
board dts file.
Since RTC interrupt line is connected to external IRQ1, so the IRQ
number of RTC should be 2 in dts.

Signed-off-by: Yang Shi <yang.shi@windriver.com>
---
 arch/powerpc/boot/dts/p1022ds.dts |    6 ++++++
 1 files changed, 6 insertions(+), 0 deletions(-)

diff --git a/arch/powerpc/boot/dts/p1022ds.dts b/arch/powerpc/boot/dts/p1022ds.dts
index 83a0111..a7c2c47 100644
--- a/arch/powerpc/boot/dts/p1022ds.dts
+++ b/arch/powerpc/boot/dts/p1022ds.dts
@@ -151,6 +151,12 @@
 			interrupts = <43 2>;
 			interrupt-parent = <&mpic>;
 			dfsrr;
+			rtc@68 {
+				compatible = "dallas,ds3232";
+				reg = <0x68>;
+				interrupts = <2 0x1>;
+				interrupt-parent = <&mpic>;
+			};
 		};
 
 		serial0: serial@4500 {
-- 
1.6.5.2

